### PR TITLE
Multi-OS Support; hidden password input

### PR DESCRIPTION
**Pull request by lewismunday**
- I have changed the clear terminal command to a function depending on OS (I would appreciate if Windows could be tested). 
- Using the getpass module, password inputs are now hidden from the screen (will only work running via the terminal, not shell).
- Changed some 'if !=' statements to 'if not'
- General file reformat for spacing and readibility

### CODE REVIEW

**Review by Abhijeetbyte — CHANGES_REQUESTED** [review]
Hi, Lewis

I appreciate your efforts, you have added some really useful features, and I have tested your changes on windows machine, clear() function works fine. Although :
* You have removed `os.system('color')`, this is an essential line of code to display colored text with ANSI codes.

* Did not changed `input()` to `getpass()` in Edit feature


  Everything else works great and soon, release a variant for Linux

Thank You

File: main.py [hunk 1]
```diff
@@ -161,190 +152,171 @@ def backup():
...
 while True:
-    
+
     try:
-        
-        Master_pass =(input("\n ENTER MASTER PASSWORD: "))
 
-        Master_pass = "".join([(str(ord(x)-96) if x.isalpha() else x) for x in list(Master_pass)])
-        Master_pass = format(Master_pass).replace("-","")
+        Master_pass = getpass(" ENTER MASTER PASSWORD: ")  # get master password from user
```
Comment (Abhijeetbyte) [inline main.py:178]: add an "\n"

File: main.py [hunk 2]
```diff
@@ -161,190 +152,171 @@ def backup():
...
-            if (len(show) > 1): #multiple credentials found, len = rows
+            if (len(show) > 1):  # multiple credentials found, len = rows
                 index = int(input("\n SELECT AN INDEX VALUE & PRESS ENTER : "))
             else:
-                index= show.index.values #take default index
+                index = show.index.values  # take default index
                 index = int(index)
 
             new_name = input("\n ENTER NEW NAME/USERNAME: ")
             new_password = input(" ENTER NEW PASSWORD: ")  # this will be encrypted
```
Comment (Abhijeetbyte) [inline main.py:271]: add a "\n"

**Comment by Abhijeetbyte** [comment]
Hi, Lewis

kindly made the requested changes so, I can merge your pull request

**Comment by lewismunday** [comment]
Hello, 

Thanks for your patience and review; I have been on holiday. I made the changes as per suggestion. Please take a moment to review

**Review by Abhijeetbyte — APPROVED** [review]

**Comment by Abhijeetbyte** [comment]
Thank you Lewis for your contribution to this repository. I'm glad to merge your pull request

**Pull request merged**